releasenotes/notes/add-rabbitmq-multiple-versions-7c6fdb470092409b.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Adds alternative RabbitMQ container images for versions 3.11, 3.12 and
+    3.13. This allows us to perform intermediary RabbitMQ upgrades prior to a
+    SLURP upgrade to Caracal. See the Kolla docs for more details:
+    https://docs.openstack.org/kolla-ansible/latest/reference/message-queues/rabbitmq.html#slurp
```

Comment:
Do these images need to be added to `stackhpc_pulp_images_kolla`?

Reply:
If that's needed for them to sync to a client's pulp, then yes

Reply:
https://github.com/stackhpc/stackhpc-kayobe-config/pull/1258